core/src/main/java/org/aerogear/mobile/core/metrics/publisher/LoggerMetricsPublisher.java

```diff
@@ -18,7 +19,7 @@ public LoggerMetricsPublisher(final Logger logger) {
     }
 
     @Override
-    public void publish(final Metrics... metrics) {
+    public void publish(MetricsPublisherListener listener, final Metrics... metrics) {
```

Comment:
There is no real reason but usually, the callback is the last parameter. 

PS: For sure some is going to ask you to make it final, so....

Reply:
I know but so do varargs (metrics), and that's mandatory. We would have to change it into a list of some sort.